### DISCUSSION TITLE
Question about Multi-Track Audio Streaming with SRT

### DISCUSSION THREAD

**Discussion by abenmohamed** [bluenviron/mediamtx#2593] [Questions]
### Question

I am attempting to achieve multi-track audio streaming using the SRT (Secure Reliable Transport) protocol in the MediaMTX project. However, I've encountered an issue where I am only able to receive a single audio track despite my efforts.

To provide more context, here are the details:

Server Version: I am using MediaMTX v1.2.0 as my server software.

Publishing with SRT: I am publishing the stream from OBS Studio (version 29) using a custom output configuration with multi-track audio enabled. OBS is configured to use the SRT protocol for streaming.

Reading from the Server: I am trying to receive the stream from the server, but I'm experiencing issues with receiving only a single audio track.

**Comment by aler9** [comment]
Hello, OBS Studio supports streaming a single video track and a single audio track at a time, while it supports recording multiple audio tracks at the same time. This is confirmed in multiple threads, like:

https://www.reddit.com/r/VIDEOENGINEERING/comments/xnee5g/streaming_an_srt_with_multiple_tracks_of_audio/

You can request this feature to OBS maintainers.

[a comment by an automated account is not shown]
